### PR TITLE
[zookeeper] set event.module and event.dataset

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?
This PR adds event.module and event.dataset to the integration.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

### CODE REVIEW

**Review by mtojek — APPROVED** [review]
LGTM

File: packages/zookeeper/data_stream/connection/fields/base-fields.yml [hunk 1]
```diff
@@ -1,12 +1,11 @@
-- name: data_stream.type
```
Comment (mtojek) [inline packages/zookeeper/data_stream/connection/fields/base-fields.yml:1]: Hey @kaiyan-sheng , I just noticed what happened here (sorry I missed it earlier). All `data_stream.*` fields should stay as is. Hence the PR job failed for: https://github.com/elastic/package-storage/pull/1517 .

Comment (mtojek) [reply packages/zookeeper/data_stream/connection/fields/base-fields.yml:1]: Oh, I see you're OOO. I can fix it, don't worry ;)

Comment (kaiyan-sheng) [reply packages/zookeeper/data_stream/connection/fields/base-fields.yml:1]: Sorry about that!! Thank you!!!

**Pull request merged**